README.md

```diff
@@ -130,6 +132,17 @@ The following image types are currently available via the `--type` argument:
 | `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 | `iso`                 | An unattended Anaconda installer that installs to the first disk found.               |
 
+## 💾 Target architecture
+
+Specify the target architecture of the system on which the disk image will be installed on. By default,
```

Comment:
I wonder if we should slip in the word "experimental" here somewhere but then we print it on the commandline and while this needs more real world testing we are somewhat confident that the feature will work fine :)

Reply:
Nevermind, this is no longer experimental it now used for a while.